platform/src/main/java/org/stellar/anchor/platform/utils/TransactionHelper.java

```diff
@@ -66,10 +66,18 @@ static GetTransactionResponse toGetTransactionResponse(
       refunds = toRefunds(txn.getRefunds(), txn.getAmountInAsset());
     }
 
+    String sender = DEPOSIT.kind.equals(txn.getKind()) ? txn.getToAccount() : txn.getFromAccount();
+
     String amountInAsset = makeAsset(txn.getAmountInAsset(), assetService, txn);
     String amountOutAsset = makeAsset(txn.getAmountOutAsset(), assetService, txn);
     String amountFeeAsset = makeAsset(txn.getAmountFeeAsset(), assetService, txn);
     String amountExpectedAsset = makeAsset(null, assetService, txn);
+    Customers.CustomersBuilder builder =
+        Customers.builder().sender(StellarId.builder().account(sender).build());
```

Comment:
This is too verbose. Can we use constructor instead of builders for simple classes?